web-api/src/persistence/postgres/polling/saveRequestResponse.ts

```diff
@@ -0,0 +1,33 @@
+import { formatNow, FORMATS } from '@shared/business/utilities/DateHandler';
+import { pgInsertInto } from '../utils/operation/pgInsertInto';
+
+
+export const saveRequestResponse = async ({
+  requestId,
+  userId,
+  chunk,
+  index,
+  totalNumberOfChunks,
+}: {
+  requestId: string;
+  userId: string;
+  chunk: string;
+  index: number;
+  totalNumberOfChunks: number;
+}) => {
+  const nowUnix = Number(formatNow(FORMATS.UNIX_TIMESTAMP_SECONDS));
+  const ttl = nowUnix + 16 * 60;
```

Comment:
Then you can use the method you exported:

```suggestion
  const ttl = getTtl();
```